### ISSUE TITLE
Feature Request - Highlight Required Fields/Dependencies

### ISSUE THREAD

**Issue by ebraker** [ArctosDB/arctos#6615]
It is easy to save records and even upload them without required information. 

Can we bring back some sort of highlight/trigger that highlights all dependencies once information is entered into any of the locality stack fields? I just had to delete a record because half the info I successfully saved in the Data Entry screen was lost during the upload stage and there was not a way to enter it once loaded. Had it not been a record that I needed to attach media to, I would have never known anything was amiss since I never received any error message.

**Comment by dustymc** [comment]
Yes, I can implement any sort of rules, but they need to be clearly defined and they will change how data may be loaded (eg lots of folks make a habit or leaving mostly-static partial data in spreadsheets so it's there when it's needed and ignored until some "upstream" bit is supplied). 

Currently the check is pass or fail, I think changing that to allow a 'warning' state (which notifies people but doesn't prevent load) would do what you're asking. I think PG has what's needed to change that, it would just be some development.

**Comment by ebraker** [comment]
Sounds good. In my case I had event, locality, and spatial data entered but did not have anything in record-event fields. Record-event should be highlighted/trigger a warning any time info is added into any of the place/time stack. 

I also think that child fields should highlight once the parent is selected and can follow the old model, e.g. I select a value from the coordinate_lat_long_units  menu and immediately datum, coordinate_georeference_protocol and the relevant coordinate fields should turn yellow. I enter a part_name, and part_disposition and part_condition fields should turn yellow. etc.
